test/testcafe/spec/TerminalView_spec.js

```diff
@@ -0,0 +1,47 @@
+import { RequestMock } from 'testcafe';
```

Comment:
Should we name this test file something specific to email? `EmailTerminalView`?

Reply:
The idea is to expand it to other type of authenticators' terminalviews by checking the i18n keys (and thus expand the same test file) OR cleaning up this code if we decide to have a proper remediation response that indicates what this terminalview corresponds to..

Reply:
Having separate test files seems like an overkill at this point..

Reply:
ok, im fine either way. Just had to ask because the tests for safe mode that use this terminal view, are in a separate safe mode test spec.